ts/src/huobi.ts

```diff
@@ -4906,42 +4908,68 @@ export default class huobi extends Exchange {
                 }
                 params = this.omit (params, [ 'client-order-id', 'client-order-ids', 'clientOrderId', 'clientOrderIds' ]);
             }
-            method = 'spotPrivatePostV1OrderOrdersBatchcancel';
+            response = await this.spotPrivatePostV1OrderOrdersBatchcancel (this.extend (request, params));
```

Comment:
@Dan-krm I think I found the reason, this should be a string array instead of a string 
![image](https://user-images.githubusercontent.com/43336371/236805583-9df12e6f-eb65-43f7-84db-44bd6c8095aa.png)